static/js/components/widgets/ResourcePickerDialog.test.tsx

```diff
@@ -102,12 +102,12 @@ describe("ResourcePickerDialog", () => {
     },
     {
       mode:         RESOURCE_LINK,
-      contentNames: ["pages"],
+      contentNames: ["page"],
```

Comment:
I think in this case:
- `"pages"` was not a valid content name
- so after filtering the array was empty, so you got that error about "Cannot read property of undefined"